pkg/metadata/metadata.go

```diff
@@ -40,13 +40,14 @@ type NodeMetadata interface {
 	GetWorkerID() string
 }
 
-type nodeMetadataManager struct {
-	zone     string
-	region   string
-	workerID string
+// NodeMetadataManager ...
+type NodeMetadataManager struct {
+	Zone     string
```

Comment:
The methods below anyway return these fields. Do these fields also need to be capitalized ?

Reply:
Looks like, this is needed for UTs. got clarified from Ambika.

Reply:
Discussed with Guna and decided to keep same code as the change was relevant for UT